app/assets/stylesheets/dashboard.scss

```diff
@@ -182,6 +182,24 @@
         font-size: 0.6em;
         vertical-align:0.16em;
       }
+      form {
+        display: inline-block;
+        background: $light-gray;
+        padding: 8px 15px;
+        input {
+          font-size: 20px;
+          border-radius: 3px;
+          padding: 3px 8px;
+          width: 50px;
```

Comment:
```suggestion
          width: 75px;
```

🤷‍♂️ 

![form width](https://user-images.githubusercontent.com/17884966/49311647-b355d100-f4af-11e8-8b69-e5a369a2ff1e.png)

Also looks funny in mobile.